cli/cmd/agent_list.go

```diff
@@ -207,17 +210,31 @@ func machineExist(machines []api.AgentInfo, mid int) bool {
 func agentInfoToListAgentTable(machines []api.AgentInfo) [][]string {
 	out := [][]string{}
 	for _, m := range machines {
+
+		// @afiune testing how to truncate a long value
+		hostname := m.Hostname
+		if len(hostname) > 20 {
+			hostname = fmt.Sprintf("%s...", hostname[:20])
```

Comment:
Are we truncating for formatting purposes? After we've obscured information how does the user see the full value, the json response?

Reply:
If we are going to include fixed field limits we should have a single function that is shared for consistency. If it's possible to include it in the tablewriter that would be nice.